sdmetrics/reports/multi_table/base_multi_table_report.py

```diff
@@ -98,6 +98,9 @@ def get_visualization(self, property_name, table_name=None):
             plotly.graph_objects._figure.Figure
                 The visualization for the requested property.
         """
+        if property_name == 'Data Structure':
+            return self._properties[property_name].get_visualization(table_name)
```

Comment:
Is this code path tested?  Codecov is reporting a miss.

Reply:
Yes, done in [fb67bc6](https://github.com/sdv-dev/SDMetrics/pull/513/commits/fb67bc651bdc87c4517a02dc9df75cad92f4ae8b)